theano/gpuarray/opt.py

```diff
@@ -56,18 +61,43 @@
 
 gpu_optimizer = EquilibriumDB()
 gpu_cut_copies = EquilibriumDB()
+gpu_topo = LocalGroupDB()
+
+# Not used for an EquilibriumOptimizer. It has the "tracks" that we need for GraphToGPUDB.
+gpu_optimizer2 = EquilibriumDB()
+
+
+class GraphToGPUDB(DB):
+    """
+    Retrieves the list local optimizers based on the optimizer flag's value
+    from EquilibriumOptimizer by calling the method query.
+
+    """
+
+    def query(self, *tags, **kwtags):
+        opt = gpu_optimizer2.query(*tags, **kwtags)
+        return GraphToGPU(opt.local_optimizers_all, opt.local_optimizers_map)
+
 
 gpu_seqopt = SequenceDB()
 
 # Don't register this right now
 conv_groupopt = LocalGroupDB()
 conv_groupopt.__name__ = "gpua_conv_opts"
 
+gpu_seqopt.register('gpuarray_graph_optimization', GraphToGPUDB(), -0.5,
+                    'fast_compile', 'fast_run', 'gpuarray')
+
 gpu_seqopt.register('gpuarray_local_optimiziations', gpu_optimizer, 1,
                     'fast_compile', 'fast_run', 'gpuarray')
 gpu_seqopt.register('gpuarray_cut_transfers', gpu_cut_copies, 2,
                     'fast_compile', 'fast_run', 'gpuarray')
 
+gpu_seqopt.register('op_lifter_topo', TopoOptimizer(gpu_topo.query('+fast_compile', '+fast_run', '+cudnn',
```

Comment:
Also, is this is the correct way to retrieve all the local optimizers in that DB